drivers/platform/stm32/stm32_i2c.c

```diff
@@ -95,8 +97,13 @@ int32_t stm32_i2c_init(struct no_os_i2c_desc **desc,
 	};
 
 	xdesc->hi2c.Instance = base;
+#if defined (STM32F4)
 	xdesc->hi2c.Init.ClockSpeed = param->max_speed_hz;
 	xdesc->hi2c.Init.DutyCycle = I2C_DUTYCYCLE_2;
+#else
+	xdesc->hi2c.Init.Timing = i2cinit->i2c_timing;
+#endif
+	stm32_calculate_i2c_timing(param->max_speed_hz, hi2c);
```

Comment:
where is this defined? I suppose you forgot to add it in the commit

Reply:
This function is not required. It was added by mistake. I've corrected the file now